draft-ietf-mls-architecture.md

```diff
@@ -604,13 +627,35 @@ states, breaking their ability to communicate.
 > Avoid using inconsistent access control policies in the case of
 > encrypted group operations.
 
+MLS allows actors outside the group to influence the group in two ways: External
+signers can submit proposals for changes to the group, and new joiners can use
+an external join to add themselves to the group.  The `external_senders`
+extension ensures that all members agree on which signers are allowed to send
+proposals, but any other policies must be assured to be consistent as above.
+When external joins are allowed in a group, a joiner cannot be vetted before
+joining, so policy enforcement must be reactive.
+
+> ** RECOMMENDATION:**
+> Have an explicit group policy about whether external joins are allowed.
+
+> **RECOMMENDATION:**
+> In a group that allows external join, verify that new joiners are authorized,
+> and remove them if not.
+
 ## Recovery After State Loss
 
 Group members whose local MLS state is lost or corrupted can reinitialize their
 state by re-joining the group as a new member and removing the member
 representing their earlier state.  An application can require that a client
 performing such a reinitialization prove its prior membership with a PSK.
 
+There are a few costs to this approach.  If a group uses such PSKs as
+the sole form of authorization to join the group, there is a risk that leaked
+PSKs could allow an unauthorized party to join.  At a practical level, the
+application will need to ensure that the new members have the required PSK,
+including any new members that have joined the group since the epoch in which
+the PSK was issued.
```

Comment:
The second sentence seems like a red herring since an external add-remove combo checks that the new leaf has an equivalent credential as the old leaf. Yes you *could* do what you say, but the first paragraph indicates that we're not. I would just keep the third sentence, which is a very good point.